apps/web/src/services/auth/password/PasswordPage.vue

```diff
@@ -12,6 +13,12 @@ import { useI18n } from 'vue-i18n';
 import { useRouter } from 'vue-router';
 import { useStore } from 'vuex';
 
+
+
+import { SpaceRouter } from '@/router';
```

Comment:
nitpick: `SpaceRouter` should be replaced to `const router = useRouter()` in `.vue` components.